### PR TITLE
[KYUUBI #2159][FLINK] Prevent loss of exception message line separator

### PR DESCRIPTION
**Pull request by deadwind4**
### _Why are the changes needed?_

Flink SQL Exception message loses line separator.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by pan3793** [comment]
Except for this one, are there other properties look unreasonable?

**Review by yaooqinn — APPROVED** [review]

**Comment by deadwind4** [comment]
I will introduce the white list in this PR.

File: kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala [hunk 1]
```diff
@@ -99,9 +99,11 @@ class FlinkProcessBuilder(
     ("FLINK_SQL_ENGINE_JAR" -> mainResource.get) +
     ("FLINK_SQL_ENGINE_DYNAMIC_ARGS" ->
       conf.getAll.map { case (k, v) =>
-        k match {
-          case "line.separator" => ""
-          case _ => s"-D$k=$v"
+        if (k.startsWith("kyuubi.") || k.startsWith("flink.")
```
Comment (pan3793) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala:102]: We can use `conf.getAll.filter{ ... } .map { ... }.mkString(" ")` to make it more readable

**Review by pan3793 — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**